includes/trackLayout2.html

```diff
@@ -1,21 +1,20 @@
 <button class="trackReturn" data-track="2">Return</button>
 <svg width="1000" height="800">
-    <rect x="0" y="0" width="1000" height="800"/>
-    <rect x="0" y="10" width="250" height="60" class="path" />
-    <rect x="250" y="10" width="50" height="220" class="path" />
-    <rect x="50" y="190" width="240" height="40" class="path" />
-    <rect x="50" y="200" width="45" height="550" class="path" />
-    <rect x="50" y="700" width="350" height="50" class="path" />
-    <rect x="350" y="250" width="50" height="500" class="path" />
-    <rect x="350" y="220" width="300" height="50" class="path" />
-    <rect x="400" y="600" width="240" height="30" class="path" />
-    <rect x="600" y="630" width="250" height="30" class="path" />
-    <rect x="850" y="100" width="60" height="650" class="path" />
-    <rect x="620" y="100" width="250" height="50" class="path" />
-    <rect x="620" y="120" width="50" height="150" class="path" />
-    <rect x="850" y="700" width="150" height="50" class="path" />
-    <rect x="0" y="10" width="60" height="60" class="finish" />
-    <rect x="950" y="700" width="50" height="50" class="finish" />
+    <rect x="0" y="0" width="1000" height="800"></rect>
+    <rect x="0" y="10" width="250" class="path" height="100"></rect>
+    <rect y="10" height="220" class="path" width="100" x="205"></rect>
```

Comment:
the rects need to be in a consistent format: 
x, y, width, height, class

at the moment these are not matching, which reduces readability